src/main/java/use_cases/login_leaderboard/LoginUser.java

```diff
@@ -2,8 +2,6 @@
 
 import entities.login_leaderboard.User;
 
-import javax.swing.*;
-
```

Comment:
Awesome. we should check for unused imports/variables/methods in other classes we have!